deepset_cloud_sdk/service/files_service.py

```diff
@@ -18,6 +20,15 @@
 logger = structlog.get_logger(__name__)
 
 
+@dataclass
+class DeepsetCloudFiles:
```

Comment:
should be the singular form DeepsetCloudFile

Reply:
Good catch!